lib/Fmt_odoc.ml

```diff
@@ -209,44 +209,39 @@ let at = char '@'
 
 let space = fmt "@ "
 
-let fmt_tag_see c wrap sr txt =
-  at $ fmt "see@ "
-  $ wrap (str sr)
-  $ fmt_nestable_block_elements c ~prefix:space txt
+let fmt_tag_args ?arg ?txt c tag =
+  at $ str tag
+  $ opt arg (fun x -> char ' ' $ x)
+  $ opt txt (fun x -> fmt_nestable_block_elements c ~prefix:space x)
```

Comment:
I wonder if we should wrap the `txt` into a box to break between the `arg` and `txt` in priority. For example, with a margin of 20:
```
(** @param Foo Bar
      Baz *)
```
could be:

```
(** @param Foo
      Bar Baz *)
```

The word `Foo` is the `arg` and `Bar Baz` is the `txt`.

Reply:
That would be a nice improvement indeed, we can do that in a follow-up PR